src/components/Thumbnail/Description.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+
+export interface IProps {
+  classname?: string;
+}
+
+export const Description: FC<IProps> = ({ children, classname = "" }) => (
```

Comment:
we don't need now default value for classname because classnames library fix this